documents/mrjones2014-legendary-nvim.md

```diff
@@ -0,0 +1,448 @@
+---
+name: mrjones2014/legendary.nvim
+slug: mrjones2014-legendary-nvim
+category: Keybinding
+created: 'Mar 12, 2022 11:35'
+description: "\U0001F5FA️ A legend for your keymaps, commands, and autocmds, with which-key.nvim integration (requires Neovim nightly)"
+url: 'https://github.com/mrjones2014/legendary.nvim'
+stars: 210
+topics:
+  - keybindings
+  - keymap
+  - keymapping
+  - legend
+  - lua
+  - neovim
+  - neovim-plugin
+  - neovim-ui
+  - nvim
+  - vim
+  - vim-commands
+  - which-key
+updated_at: '2022-03-12T20:58:52Z'
+---
+# Legendary
+
+Define your keymaps, commands, and autocommands as simple Lua tables, building a legend at the same time.
+
+<!-- panvimdoc-ignore-start -->
+
+![demo](./demo.gif) <sup>Theme used in recording is [lighthaus.nvim](https://github.com/mrjones2014/lighthaus.nvim). The finder UI is handled by [telescope.nvim](https://github.com/nvim-telescope/telescope.nvim) via [dressing.nvim](https://github.com/stevearc/dressing.nvim). See [Prerequisites](#prerequisites) for details.</sup>
```

Comment:
is this gif URL going to resolve?

Reply:
It doesn't in the preview build, going to figure out some solution. I appreciate you pointing that out.

Reply:
I can just update it to use an absolute URL if that's easier.

Reply:
That'd be great, you're legendary.

I certainly don't want that to be a requirement for submitting something though, as it seems its pretty common to use a relative path for media in READMEs. I'm letting a library handle all of the markdown to HTML converting and then just injecting the HTML that way. 

I can potentially write something that parses the markdown for assets, and checks if it's a path instead of a URL, but seems like it might take a bit. I would really value your ideas if you have anything in mind.

Reply:
No problem. I've updated it to an absolute URL.

You could essentially replace the path with `https://github.com/mrjones2014/legendary.nvim/raw/master/[rest of path here]`, but you'd have to parse the markdown and get each link. Not sure how you'd do that. Tbh it might be more effort than it's worth ¯\_(ツ)_/¯

Reply:
lol agreed.